ios/TestApp/TestApp/ViewController.mm

```diff
@@ -1,5 +1,5 @@
 #import "ViewController.h"
-#import <LibTorch/LibTorch.h>
```

Comment:
are you going to delete the header file?

Reply:
No, the umbrella header is installed via cocoapods, which is not needed in this case. I replaced it with `import<torch/script.h>`